interim-19-01/agenda.md

```diff
@@ -49,6 +49,9 @@ _~15 minutes_
 - Next interop
 - Getting done
 
+### Remote presentations
```

Comment:
Please just put the bullet under "As Time Permits" - no reason for a separate section.